sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -3848,7 +3848,6 @@ def idxmin(self, **kwargs):
   def idxmax(self, **kwargs):
     return self._idxmaxmin_helper('idxmax', **kwargs)
 
-
```

Comment:
Unrelated change. Remove?

Reply:
Done. Sorry, something happened with the formatter.